synthtool/gcp/templates/java_library/.kokoro/nightly/integration.cfg

```diff
@@ -13,12 +13,12 @@ env_vars: {
 # TODO: remove this after we've migrated all tests and scripts
 env_vars: {
   key: "GCLOUD_PROJECT"
-  value: "gcloud-devel"
+  value: "java-docs-samples-testing"
```

Comment:
@Neenu1995 Was this intentional? It's causing failures in java-spanner-jdbc. https://github.com/googleapis/java-spanner-jdbc/issues/778